rollup.config.js

```diff
@@ -21,6 +21,7 @@ module.exports = [
   makeConfigForOutput({
     file: pkg.main,
     format: 'umd',
-    name: 'contentfulExtension'
+    name: 'contentfulExtension',
+    footer: 'window.contentfulApp = window.contentfulExtension;'
```

Comment:
Nice, `footer`. How about `;window.contentfulApp....`?

Reply:
As we run terser over the whole thing, the minified JavaScript is identical.

The footer is inserted after the wrapping function which already has a semicolon at its end.

```javascript
[...]
    };
}var init = createInitializer(window, createAPI);exports.init=init;exports.locations=locations;Object.defineProperty(exports,'__esModule',{value:true});})));
window.contentfulApp = window.contentfulExtension;
```